src/form-builder/components/ScriptEditorModal.js

```diff
@@ -2,18 +2,51 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import NotificationContainer from 'common/Notification';
 import { commonConstants } from 'common/constants';
+import CodeMirror from 'codemirror';
+import { JSHINT } from 'jshint';
+import 'codemirror/mode/javascript/javascript';
+import 'codemirror/lib/codemirror.js';
+import 'codemirror/lib/codemirror.css';
+import 'codemirror/mode/javascript/javascript.js';
+import 'codemirror/addon/lint/lint.js';
+import 'codemirror/addon/lint/lint.css';
+import 'codemirror/addon/lint/javascript-lint.js';
+import jsBeautifier from 'js-beautify';
+import 'codemirror/addon/hint/show-hint.js';
+import 'codemirror/addon/hint/javascript-hint.js';
+import 'codemirror/addon/edit/closebrackets.js';
 
+window.JSHINT = JSHINT;
```

Comment:
where is this variable used? can't we add global variables like this?

Reply:
Code Mirror needs this to be declared in the window object.

Reply:
We should add it as a comment, else no one will know,